client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/hooks/use-product-and-plans.tsx

```diff
@@ -64,6 +64,33 @@ const getProductsAndPlansByFilter = (
 	return allProductsAndPlans || [];
 };
 
+// This function gets the displayable Plans based on how it should be arranged in the listing.
+const getDisplayablePlans = ( filteredProductsAndBundles: APIProductFamilyProduct[] ) => {
+	const plans = getProductsAndPlansByFilter( PRODUCT_FILTER_PLANS, filteredProductsAndBundles );
+
+	const securityPlans = plans.filter( ( { slug } ) => slug.startsWith( 'jetpack-security' ) );
+	const restOfPlans = plans.filter( ( { slug } ) => ! slug.startsWith( 'jetpack-security' ) );
+
+	return [ securityPlans, ...restOfPlans ];
+};
+
+// This function gets the displayable Products based on how it should be arranged in the listing.
+const getDisplayableProducts = ( filteredProductsAndBundles: APIProductFamilyProduct[] ) => {
+	const products = getProductsAndPlansByFilter(
+		PRODUCT_FILTER_PRODUCTS,
+		filteredProductsAndBundles
+	);
+
+	const backupProducts = products.filter( ( { slug } ) => slug.startsWith( 'jetpack-backup' ) );
+	const restOfProducts = products.filter( ( { slug } ) => ! slug.startsWith( 'jetpack-backup' ) );
+
+	return [ ...restOfProducts, backupProducts ].sort( ( a, b ) => {
+		const product_a = Array.isArray( a ) ? a[ 0 ].name : a.name;
+		const product_b = Array.isArray( b ) ? b[ 0 ].name : b.name;
+		return product_a.localeCompare( product_b );
+	} );
+};
```

Comment:
I think we can make it more generic and scalable rather than restricting it to only filter by `jetpack-backup`. WDYT?

```suggestion
const PRODUCT_FILTERS = [ 'jetpack-backup' ];

const getDisplayableProducts = ( filteredProductsAndBundles: APIProductFamilyProduct[] ) => {
	const products = getProductsAndPlansByFilter(
		PRODUCT_FILTER_PRODUCTS,
		filteredProductsAndBundles
	);

	const filteredProducts = PRODUCT_FILTERS.map( ( filter ) => {
		return products.filter( ( { slug } ) => slug.startsWith( filter ) );
	} ).filter( ( subArray ) => subArray.length > 0 ); // Remove empty arrays

	const restOfProducts = products.filter( ( { slug } ) => {
		return ! PRODUCT_FILTERS.some( ( filter ) => slug.startsWith( filter ) );
	} );

	return [ ...restOfProducts, ...filteredProducts ].sort( ( a, b ) => {
		const product_a = Array.isArray( a ) ? a[ 0 ].name : a.name;
		const product_b = Array.isArray( b ) ? b[ 0 ].name : b.name;
		return product_a.localeCompare( product_b );
	} );
};
```

Reply:
Great suggestion! 👍